packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -246,10 +272,15 @@ class ChatMessage extends UIComponent<WithAsProp<ChatMessageProps>, ChatMessageS
     })
 
     return (
-      <Ref innerRef={this.messageRef}>
+      <Ref
+        innerRef={domNode => {
+          !this.state.messageDomNode && this.setState({ messageDomNode: domNode })
+        }}
+      >
         <ElementType
           onBlur={this.handleBlur}
           onFocus={this.handleFocus}
+          onMouseEnter={() => this.updateActionsMenuPosition()}
```

Comment:
is it ok not to update the position when you are only using keyboard?

Reply:
yes, as logic of `this.handleFocus` is responsible for making updates in this case: